scripts/create-aks-cluster.sh

```diff
@@ -25,8 +25,6 @@ main() {
   if [[ "$(should_create_aks_cluster)" == "true" ]]; then
     echo "Creating an AKS cluster '${CLUSTER_NAME}'"
     LOCATION="$(get_random_region)"
-    # pin to the minor version and aks will pick the latest patch version
-    KUBERNETES_VERSION="1.26"
```

Comment:
We have a test matrix with kind clusters, so relying on default version for AKS cluster creation should be fine.